tools/circt-opt/circt-opt.cpp

```diff
@@ -5,11 +5,17 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "circt/Conversion/LLHDToLLVM/LLHDToLLVM.h"
+#include "circt/Conversion/StandardToHandshake/StandardToHandshake.h"
 #include "circt/Dialect/FIRRTL/Dialect.h"
-#include "circt/Dialect/RTL/Dialect.h"
 #include "circt/Dialect/Handshake/HandshakeOps.h"
-#include "circt/Conversion/StandardToHandshake/StandardToHandshake.h"
+#include "circt/Dialect/LLHD/IR/LLHDDialect.h"
+#include "circt/Dialect/LLHD/Transforms/Passes.h"
+#include "circt/Dialect/RTL/Dialect.h"
+#include "mlir/Conversion/StandardToLLVM/ConvertStandardToLLVMPass.h"
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
 #include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/IR/AsmState.h"
```

Comment:
Are all these #includes necessary?  AsmState.h?

Reply:
I included `ASMState.h` to get access to the `registerAsmPrinterCLOptions();` function, which adds the `--mlir-print-op-generic` command line option. We recently started using it in our roundtrip test cases to make them easier to read, by having the custom syntax also in the input. I'm not sure if there would be another way to get access to that function without including the header. 
On the other hand registering the `ConvertStandardToLLVM` was indeed unnecessary (removed in f38464a02db2943d146361b90ed566209572a9f9).

Reply:
Sounds good, thanks!